packages/webapp/src/components/TileDashboard/MoreComponent/index.tsx

```diff
@@ -62,7 +64,7 @@ export const MoreComponent = ({
                 <DashboardTile
                   key={index}
                   {...item}
-                  isSelected={selectedFilterIds?.includes(item.id)}
+                  isSelected={item.isSelected ?? selectedFilterIds?.includes(item.id)}
```

Comment:
I'm so sorry 🙇 This was my sheer laziness -- `isSelected` was never meant to be passed within the item data array, I only (and this is really bad, I am a little surprised I did this) put it on the TypeCountTile interface... in order to recycle that interface on the DashboardTile 😬🙇 

What I should have done is remove `isSelected` from `TypeCountTile` and done this:

```ts
interface DashboardTileProps extends TypeCountTile {
  isSelected: boolean;
}
```

I don't think there is a way to get the interaction to work with a data-passed `isSelected` -- but is that perhaps the purpose with the `??` -- to override the interactivity? If so it's a neat feature now!

I'm sorry about the misleading TypeScript 😞